pkg/vulnsrc/python/python.go

```diff
@@ -154,3 +156,8 @@ func (ad AdvisoryDB) UnmarshalJSON(data []byte) error {
 	}
 	return nil
 }
+func ToLowerCasePythonPackage(pkg string) string {
+	pkg = strings.ToLower(pkg)
+	pkg = strings.ReplaceAll(pkg, "_", "-")
+	return pkg
+}
```

Comment:
Could you add a commend and reference?
https://www.python.org/dev/peps/pep-0426/#name